cypress/integration/SingleSelect/duplicate_option_values.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Dealing with duplicate option values inconsistently
```

Comment:
Maybe you could call this: `Highlights all duplicate option values`, because even though it could be considered wrong behaviour, it's still consistent.

Reply:
The inconsistency I am referring to is the fact that only a single option is "selected" in the input, but both are "selected" in the dropdown. I suppose you are right too, in the sense that it is consistently displaying this inconsistent behaviour 🙄 ...

The problem with `Highlights all duplicate option values` is that it only focusses on the dropdown, not the input.

How about something like this:
`Duplicate option values result in a conflict in between the input and dropdown UI`

Reply:
Ah I see what you mean now, in that case your suggestion seems like a good description 👍